Modules/IO/NIFTI/test/itkNiftiLargeImageRegionReadTest.cxx

```diff
@@ -32,6 +32,12 @@ itkNiftiLargeImageRegionReadTest(int argc, char * argv[])
     return EXIT_FAILURE;
   }
 
+  if (sizeof(void *) <= 4) // 32-bit architecture
+  {
+    std::cout << "Test not run." << std::endl;
```

Comment:
Can a more informative message be used so that the user knows why the test is excluded?
e.g.
"Test not run: test data requires > 4 GB memory allocation; the detected system architecture is 32-bit, which prevents allocating the sufficient amount of memory."

Reply:
Honestly i think a shorter message that the test is disabled on 32-bit platforms were enough, without details. But it is not OK, not the reason to change the PR.